examples/quantize/quantize.cpp

```diff
@@ -76,7 +78,12 @@ void usage(const char * executable) {
     fprintf(stderr, "  --leave-output-tensor: Will leave output.weight un(re)quantized. Increases model size but may also increase quality, especially when requantizing\n");
     fprintf(stderr, "\nAllowed quantization types:\n");
```

Comment:
I think these `fprintf` should become `printf` so that help goes to `stdout`

Reply:
Should stuff like that tool be adapted to use the logging system? I see the internal quantize stuff in `llama.cpp` reports its progress with `LLAMA_LOG_INFO`, etc.

Reply:
I don't think so - it's specific to the current tool. `llama.cpp` can be used by 3rd party projects and that's why it needs the logging interface